.github/workflows/32-bit-linux.yml

```diff
@@ -17,7 +17,7 @@ permissions:
 
 jobs:
   pytest:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Please don't unpin this. You should change this to ubuntu-22.04 everywhere.

Reply:
20.04 or 22.04? This was just a git revert

Reply:
We can bump up to 22.04 if CI is green.

Reply:
xref https://github.com/pandas-dev/pandas/issues/50115